src/cli.rs

```diff
@@ -337,16 +335,20 @@ impl Config {
         };
 
         if image.is_some() && image_backend.is_none() {
-            return Err("Could not detect a supported image backend".into());
+            bail!("Could not detect a supported image backend");
```

Comment:
I don't get that one, the `Config` expects an Option for `image_backend`.

If I do `image_backend.context("No supported backend")?`,it will unwrap the inner value.

Reply:
Sorry, just realized I was pretty vague in my comment 😅 
What I meant was moving the `Some` vs `None` check from this line to where the the value in the `Option` is needed, since `Context::context`/`Context::with_context` can be used as an `is_some` check.
I'm not sure if it would be better to leave the `is_some` check here, or move it to where the value is needed to be unwrapped.

Reply:
Got it! 659712c